ProjectSettings/ProjectSettings.asset

```diff
@@ -17,7 +17,7 @@ PlayerSettings:
   defaultCursor: {fileID: 0}
   cursorHotspot: {x: 0, y: 0}
   m_SplashScreenBackgroundColor: {r: 0.13725491, g: 0.12156863, b: 0.1254902, a: 1}
-  m_ShowUnitySplashScreen: 0
+  m_ShowUnitySplashScreen: 1
```

Comment:
Change back to `0`

Reply:
i will do this next week as i do not have time this week

Reply:
Okay! and yes it is